packages/edit-post/src/store/reducer.js

```diff
@@ -238,6 +238,37 @@ export function deviceType( state = 'Desktop', action ) {
 	return state;
 }
 
+/**
+ * Reducer returning state for the save panel.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action  Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+export function entitiesSavedStates(
+	state = { isOpen: false, onRequestClose: () => {} },
+	action
+) {
+	switch ( action.type ) {
+		case 'OPEN_ENTITIES_SAVED_STATES':
+			return {
+				...state,
+				isOpen: true,
+				onRequestClose: action.onRequestClose || function() {},
```

Comment:
I tried that initially and encountered some errors.  But if we are removing the callback from the reducer (or the store changes altogether) this will be gone.

Reply:
It might be because `this` is set to the action object?